test/unit/api/style.test.js

```diff
@@ -79,6 +79,24 @@ describe('api/style', () => {
                 }).toThrowError('`resolution` must be a number.');
             });
 
+            it('should throw an error when resolution is too small', () => {
+                const styleSpec = {
+                    resolution: 0
+                };
+                expect(function () {
+                    new Style(styleSpec);
+                }).toThrowError(/[\s\S]*must.*greater[\s\S]*/g);
```

Comment:
I like to use the full string exception. `'`resolution` must be greater than $0.'`

Reply:
That repeats the entire error message two times, why coupling so much the test when it is not needed?

Reply:
The error strings are part of the API, as a function or a namespace. If you change it, the interface you provide to the user changes and we need to detect that the API has changed.

In this case if you change min or max resolution (API) no test fails. Adding the full text this case is also covered.